main.go

```diff
@@ -143,6 +157,9 @@ func ProcessFilesByDay(wh warehouse.Warehouse, tableColumns []string, fs *fullst
 	if len(exports) == 0 {
 		return 0, nil
 	}
+	if conf.Local.SaveAsJson {
+		panic("The option to process files by day is only supported for CSV format.")
```

Comment:
See the comment on `SaveSyncPoints` for more details, but I think a `log.Fatalf` is preferable to a panic here, as it will give nicer output (panic prints a huge stack trace).

Reply:
Will fix